java/ql/src/Security/CWE/CWE-266/IntentUriPermissionManipulation.ql

```diff
@@ -0,0 +1,24 @@
+/**
+ * @name Intent URI permission manipulation
+ * @description Returning an externally provided Intent via setResult may allow a malicious
```

Comment:
I like the reworded description 👍🏻 😃 
As per the guidance [here](https://github.com/github/codeql/blob/main/docs/query-metadata-style-guide.md#query-descriptions-description)
```suggestion
 * @description Returning an externally provided Intent via 'setResult' may allow a malicious
```